app/views/ilb_mailer/failed_ilb_notification.text.erb

```diff
@@ -0,0 +1,25 @@
+On <%= l @patron_request.created_at, format: :short %>, <%= @patron_request.patron.display_name %> <<%= @patron_request.patron.email %>> requested the following:
+
+https://searchworks.stanford.edu/view/<%= @patron_request.instance_hrid %>
+
+However, there was a problem with the ILLiad API request:
+
+<%= @illiad_response['Message'] %>
```

Comment:
I tried to hack this to force this to pass me an error message.  Is it possible that illiad_response could be nil? B/c if so, this will throw an error. Thanks.

Reply:
I'm not sure, but it can't hurt to be defensive about it 🤷‍♂️